src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -172,32 +185,52 @@ def _version(self):
     def _list_bundles(self,
                       source: TDRSourceRef,
                       prefix: str
-                      ) -> list[AnvilBundleFQID]:
+                      ) -> list[TDRAnvilBundleFQID]:
         spec = source.spec
         partition_prefix = spec.prefix.common + prefix
         validate_uuid_prefix(partition_prefix)
         primary = BundleEntityType.primary.value
         supplementary = BundleEntityType.supplementary.value
-        rows = self._run_sql(f'''
+        dataset = BundleEntityType.duos.value
```

Comment:
```suggestion
        duos = BundleEntityType.duos.value
```

at least that's what the existing pattern seems to suggest.